$NetBSD: patch-ai,v 1.3 1999/09/29 20:38:28 hubertf Exp $

This helped me getting kdm going on Solaris 7/x86. I did not test this
with Zoularis, but maybe it'll help if someone needs it.

--- kdm/daemon.c.orig	Wed Sep 30 23:19:53 1998
+++ kdm/daemon.c	Fri Sep 24 16:58:00 1999
@@ -38,6 +38,10 @@
 
 #include <X11/Xos.h>
 
+#ifdef __SVR4
+#define SVR4 __SVR4
+#endif
+
 /*
 #if defined(SVR4) || defined(USG)
 #include <termios.h>
@@ -135,14 +139,12 @@
     /*
      * Close standard file descriptors and get rid of controlling tty
      */
-
-/*
 #if defined(SYSV) || defined(SVR4)
     setpgrp ();
 #else
     setpgrp (0, getpid());
 #endif
-*/
+
     setpgid( 0, getpid()); /* This should be POSIX /stefh */
     close (0); 
     close (1);
